src/pages/Dashboard/components/AccountList/AccountTable.tsx

```diff
@@ -43,18 +53,52 @@ const emptyContainerProps: FlexProps = {
 export const AccountTable = memo(() => {
   const loading = useSelector(selectIsPortfolioLoading)
   const rowData = useSelector(selectPortfolioAccountRows)
+  const assets = useSelector(selectAssets)
   const receive = useModal('receive')
   const assetActionsDrawer = useModal('assetActionsDrawer')
-  const sortedRows = useMemo(() => {
-    return rowData.sort((a, b) => Number(b.fiatAmount) - Number(a.fiatAmount))
-  }, [rowData])
+
+  const uniqueRowsAll = useMemo(() => {
+    const grouped = groupBy(rowData, row => {
+      const asset = assets[row.assetId]
+      if (!asset) return row.assetId
+      const baseAssetName = asset.name.split(' on ')[0]
+      return `${baseAssetName}_${asset.symbol}`
+    })
+
+    const groupedResults = Object.values(grouped).map(group => {
+      if (group.length === 1) return group[0]
```

Comment:
q: this applies for all occurrences of `group[0]` in this PR but am I correct in assuming we *could* run into the wrong primary asset, but that's actually not an issue, since as far as this PR is concerned, a primary asset is just used to display one of many implementations, and we're not so concerned about it being the "correct" one (e.g ETH on Ethereum) but rather it being the row?

Reply:
Not existing anymore, we now rely on the primary related asset